### DISCUSSION TITLE
Page Save/Saved Decoration

### DISCUSSION THREAD

**Discussion by arkansas-coder** [requarks/wiki#6241] [Help / Questions] [answered]
Raspberry Pi OS
Docker
Wiki.js 2.5.297

This is a minor issue / bug. 

Running two instances on the same browser, Microsoft Edge, and logged into the same account, the first instance will show "SAVE"/"SAVED" but the second instance does not. It is handy having the reminder; is there a way to force it to always appear?

**Comment by NGPixel** [comment, marked as answer]
The state is determined by whether you've made changes to the page (started editing content, changed properties, etc.), at which point it will change to "SAVE".

**Comment by arkansas-coder** [comment]
"SAVE" / "SAVED" appears to show for the first instance of Wiki.js. Unfortunately, this is not the case for other instances. The second and subsequent instances do not show either "SAVE" or "SAVED".  The image does not show SAVE or SAVED. 
Thanks for all your great work and your help. Wiki.js is truly a lifesaver, and I'm looking forward to 3.0.

![image](https://user-images.githubusercontent.com/14116217/225016000-e4445265-bdde-47cc-a3bd-c12cb4087c52.png)

**Reply by NGPixel** [reply]
That's because your browser window is too small? Maximize it to the same width as your other instance and the texts will appear...

**Comment by arkansas-coder** [comment]
Thanks, that appears to be the problem. If I open a new browser full-screen and start Wiki.js, it will not have Save/Saved. If I resize the window smaller, the text appears and will stay when I go back to full screen. I don't understand why this is the case, but my problem is solved. Thanks!
